bubble_sort.rb

```diff
@@ -0,0 +1,17 @@
+# arr = %w[a j h tu nmo ki]
+def bubble_sort(arr)
+  yield(arr)
+end
+arr = [5, 4, 3, 2, 1]
+
+bubble_sort(arr) do
+  l = arr.length - 1
+  counter = 0
+  while counter < arr.length
+    l.times do |x|
+      arr[x], arr[x + 1] = arr[x + 1], arr[x] if arr[x] > arr[x + 1]
+    end
+    counter += 1
+  end
+  print arr
+end
```

Comment:
This should be placed inside the `bubble_sort` method
Also remove the `print`,so that when you call it looks like this  `print bubble_sort([1, 2, 3, 4, 5])`